pkgs/development/python-modules/cyclonedx-python-lib/default.nix

```diff
@@ -49,6 +49,7 @@ buildPythonPackage rec {
     py-serializable
     types-setuptools
     types-toml
+    jsonschema
```

Comment:
We should start using `optional-dependencies`and follow the upstream requirements (https://github.com/CycloneDX/cyclonedx-python-lib/blob/main/pyproject.toml#L78-L81).